### ISSUE TITLE
Zoom 200% and 400% in vewers return black screens

### ISSUE THREAD

**Issue by Symbian9** [olive-editor/olive#332]
Zoom from 10% to 150% work as expected, but if set it to 200% or 400% viewers just return black screens

- Here is 150% (in both viewers)

![pic.1](https://i.imgur.com/megLqRh.png)

- Here is 200% (in both viewers)

![pic.2](https://i.imgur.com/pBEmhOy.png)

**Comment by itsmattkc** [comment]
I'm guessing this is another older GPU issue, perhaps running out of VRAM. Anything coming up in the debug log regarding this?

**Comment by Symbian9** [comment]
```
$ ./Olive-b2ff4db-x86_64.AppImage
...
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete, missing attachment.
```

**Comment by itsmattkc** [comment]
Yes I believe your computer is running out of video memory trying to draw to a larger surface. A possible optimization I could do now with the rendering done in a separate thread is increasing the size of the texture rather than the size of the surface to conserve video memory. I'll look into implementing that some day.
